[NewProduct] Add Intel processors

Evernow opened the pull request:
Sources used: 

Release date for crystal well: https://en.wikichip.org/wiki/intel/crystal_well
Release date for haswell: https://en.wikipedia.org/wiki/Haswell_(microarchitecture)
Release and discontinued date for ivy bridge: https://en.wikipedia.org/wiki/Ivy_Bridge_(microarchitecture)
Release date for devil's canyon: https://www.techpowerup.com/cpu-specs/core-i5-4690k.c1762
Discontinued date for devil's canyon: https://ark.intel.com/content/www/us/en/ark/products/80811/intel-core-i54690k-processor-6m-cache-up-to-3-90-ghz.html
Discontinued status for crystal well: https://ark.intel.com/content/www/us/en/ark/products/codename/51802/products-formerly-crystal-well.html
Release and discontinued date for broadwell: https://en.wikipedia.org/wiki/Broadwell_(microarchitecture)
Release and discontinued date for skylake: https://en.wikipedia.org/wiki/Skylake_(microarchitecture)
Release date for apollo lake: https://en.wikichip.org/wiki/intel/cores/apollo_lake
Discontinued status for bay trail: https://ark.intel.com/content/www/us/en/ark/products/93022/intel-pentium-processor-a1020-2m-cache-up-to-2-66-ghz.html
Release date for bay trail: https://www.notebookcheck.net/Intel-introduces-new-Bay-Trail-processors-for-notebooks-desktops-and-all-in-ones.101769.0.html
Release date for braswell: https://en.wikichip.org/wiki/intel/cores/braswell
I have no source for braswell or cherry tail discontinuation, but I feel it is pretty safe to assume given how old they are and no reseller for their CPUs being found. 
Release date for cherry tail: https://en.wikichip.org/wiki/intel/cores/cherry_trail
Release and discontinued date for Cannon Lake : https://en.wikipedia.org/wiki/Cannon_Lake_(microprocessor)
Avoton is simply stating the third quarter of 2013, as that is the only "date" I found: https://en.wikipedia.org/wiki/Silvermont#Server_processors_(Avoton)
Similar case with Briarwood: https://en.wikipedia.org/wiki/List_of_Intel_Atom_processors#%22Briarwood%22_(32_nm)
Release date for Centerton: https://www.anandtech.com/show/6509/intel-launches-centerton-atom-s1200-family-first-atom-for-servers
Release and discontinued date for Kaby Lake: https://en.wikipedia.org/wiki/Kaby_Lake
Release date for gemini lake: https://en.wikichip.org/wiki/intel/cores/gemini_lake
Discontinued status for gemini lake: https://ark.intel.com/content/www/us/en/ark/products/codename/83915/products-formerly-gemini-lake.html
Release and discontinued date for Kaby Lake: https://en.wikipedia.org/wiki/Coffee_Lake
Release date for Comet Lake: https://en.wikipedia.org/wiki/Comet_Lake_(microprocessor)
Release date for Ice Lake: https://en.wikipedia.org/wiki/Ice_Lake_(microprocessor)
Release date for Elkhart lake: https://www.anandtech.com/show/16102/intel-launches-10nm-atom-embedded-cpus-elkhart-lake-now-available
Discontinued status for lakefield: https://www.extremetech.com/computing/324435-intel-eols-lakefield-its-first-x86-hybrid-cpu
Release date for Lakefield: https://newsroom.intel.com/press-kits/lakefield/
Release date for Tiger Lake: https://en.wikipedia.org/wiki/Tiger_Lake
Release date for Jasper Lake: https://www.anandtech.com/show/16388/intel-launches-jasper-lake-tremont-atom-cores-for-all
Release date for Rocket Lake: https://en.wikipedia.org/wiki/Rocket_Lake

Evernow committed:
Add Intel processors

Sources used: 

Release date for crystal well: https://en.wikichip.org/wiki/intel/crystal_well
Release date for haswell: https://en.wikipedia.org/wiki/Haswell_(microarchitecture)
Release and discontinued date for ivy bridge: https://en.wikipedia.org/wiki/Ivy_Bridge_(microarchitecture)
Release date for devil's canyon: https://www.techpowerup.com/cpu-specs/core-i5-4690k.c1762
Discontinued date for devil's canyon: https://ark.intel.com/content/www/us/en/ark/products/80811/intel-core-i54690k-processor-6m-cache-up-to-3-90-ghz.html
Discontinued status for crystal well: https://ark.intel.com/content/www/us/en/ark/products/codename/51802/products-formerly-crystal-well.html
Release and discontinued date for broadwell: https://en.wikipedia.org/wiki/Broadwell_(microarchitecture)
Release and discontinued date for skylake: https://en.wikipedia.org/wiki/Skylake_(microarchitecture)
Release date for apollo lake: https://en.wikichip.org/wiki/intel/cores/apollo_lake
Discontinued status for bay trail: https://ark.intel.com/content/www/us/en/ark/products/93022/intel-pentium-processor-a1020-2m-cache-up-to-2-66-ghz.html
Release date for bay trail: https://www.notebookcheck.net/Intel-introduces-new-Bay-Trail-processors-for-notebooks-desktops-and-all-in-ones.101769.0.html
Release date for braswell: https://en.wikichip.org/wiki/intel/cores/braswell
I have no source for braswell or cherry tail discontinuation, but I feel it is pretty safe to assume given how old they are and no reseller for their CPUs being found. 
Release date for cherry tail: https://en.wikichip.org/wiki/intel/cores/cherry_trail
Release and discontinued date for Cannon Lake : https://en.wikipedia.org/wiki/Cannon_Lake_(microprocessor)
Avoton is simply stating the third quarter of 2013, as that is the only "date" I found: https://en.wikipedia.org/wiki/Silvermont#Server_processors_(Avoton)
Similar case with Briarwood: https://en.wikipedia.org/wiki/List_of_Intel_Atom_processors#%22Briarwood%22_(32_nm)
Release date for Centerton: https://www.anandtech.com/show/6509/intel-launches-centerton-atom-s1200-family-first-atom-for-servers
Release and discontinued date for Kaby Lake: https://en.wikipedia.org/wiki/Kaby_Lake
Release date for gemini lake: https://en.wikichip.org/wiki/intel/cores/gemini_lake
Discontinued status for gemini lake: https://ark.intel.com/content/www/us/en/ark/products/codename/83915/products-formerly-gemini-lake.html
Release and discontinued date for Kaby Lake: https://en.wikipedia.org/wiki/Coffee_Lake
Release date for Comet Lake: https://en.wikipedia.org/wiki/Comet_Lake_(microprocessor)
Release date for Ice Lake: https://en.wikipedia.org/wiki/Ice_Lake_(microprocessor)
Release date for Elkhart lake: https://www.anandtech.com/show/16102/intel-launches-10nm-atom-embedded-cpus-elkhart-lake-now-available
Discontinued status for lakefield: https://www.extremetech.com/computing/324435-intel-eols-lakefield-its-first-x86-hybrid-cpu
Release date for Lakefield: https://newsroom.intel.com/press-kits/lakefield/
Release date for Tiger Lake: https://en.wikipedia.org/wiki/Tiger_Lake
Release date for Jasper Lake: https://www.anandtech.com/show/16388/intel-launches-jasper-lake-tremont-atom-cores-for-all
Release date for Rocket Lake: https://en.wikipedia.org/wiki/Rocket_Lake

Evernow committed:
Use eol column

Evernow committed:
eol -> false

captn3m0 commented on products/intelprocessors.md:
```diff
@@ -0,0 +1,171 @@
+
+---
+
+> Intel is a corporation that is known for being the world's largest semiconductor chip manufacturer. 
```
Would prefer if we could reword this to point to the chips instead of the company - the page is about the former.

Evernow replied:
Maybe https://github.com/endoflife-date/endoflife.date/pull/1397/commits/65cf4030a91c97be9c7ece893ecea6c1f4bfad9c helps?

captn3m0 commented on products/intelprocessors.md:
```diff
@@ -0,0 +1,171 @@
+releaseDateColumn: true
+sortReleasesBy: 'releaseDate'
+releases:
+-   releaseCycle: "Ivy Bridge (excl. Xeon E5v2/E7v2)"
```
Let's split up releaseCycle and releaseLabels to make for a cleaner API. Unsure about using snake_case, but I think that's how Intel returns results on the CLI - Don't have a 2 word processor to check against.

```yaml
- releaseCycle: ivy_bridge
  releaseLabel: Ivy Bridge

- releaseCycle: ivy_bridge_server
  releaseLabel: Xeon E5v2/E7v2
```
Or alternatively:
```yaml
- releaseCycle: ivy_bridge_server (Or maybe ivy_bridge_xeon)
  releaseLabel: Ivy Bridge (Xeon E5v2/E7v2)
```

Evernow replied:
Did I miss something with https://github.com/endoflife-date/endoflife.date/pull/1397/commits/9b5c3aca20c3f03a4887a41047376566694f2f7d ?

captn3m0 replied:
Yeah, the remaining releases still need updating - 9b5c3aca20c3f03a4887a41047376566694f2f7d only sets it for 2.

Evernow replied:
> Yeah, the remaining releases still need updating - [9b5c3ac](https://github.com/endoflife-date/endoflife.date/commit/9b5c3aca20c3f03a4887a41047376566694f2f7d) only sets it for 2.

https://github.com/endoflife-date/endoflife.date/pull/1397/commits/40b23475b15c38069e01c845aa70a58b2448113f

I believe that catches that and another mistake

captn3m0 commented on products/intelprocessors.md:
```diff
@@ -0,0 +1,171 @@
+title: Intel Processors
+iconSlug: intel
+category: device
+versionCommand: "'msinfo32' on Windows, 'cat /proc/cpuinfo' on Linux"
```
`cpuinfo` doesn't return my processor model for me (`skylake`). `cat /sys/devices/cpu/caps/pmu_name` does work.

Evernow replied:
Changed with https://github.com/endoflife-date/endoflife.date/pull/1397/commits/9f303fee3f958a4646cbe62839b705fe0793d4bb

captn3m0 commented on products/intelprocessors.md:
```diff
@@ -0,0 +1,171 @@
+---
+
+> Intel is a corporation that is known for being the world's largest semiconductor chip manufacturer. 
+
```
Are we covering Intel Mobile CPUs completely here? For eg, I see some [CPUs here](https://compatibleproducts.intel.com/ProductDetails?activeModule=Processor%20OS%20Compatibility&prdName=Intel%C2%AE%20Core%20i7) that don't list Linux or Chrome OS - should this be covered here, or come under separate page for intel-mobile CPUs?

Evernow replied:
Speculation but I believe that when Linux OS is not listed it's when the CPU was only sold to OEMs and none of those OEMs advertise Linux support. For example the 8565U which doesn't have Linux listed on that page does seem to work fine with the stock Linux kernel: https://linux-hardware.org/index.php?id=cpu:intel-6-142-12-core-i7-8565u

captn3m0 reviewed (changes requested):
Haven't checked dates yet, few suggestions.

Evernow committed:
Change linux command

Evernow committed:
Renme Haswell/Ivy bridge xeon releases

Evernow committed:
Change description

captn3m0 committed:
Update Netlify Build dependencies (#1413)

Updates Python, Ruby, and Netlify Plugins
Keep them on the repo end, to make changes easier.
Closes #1371

Evernow committed:
Change update command

Evernow committed:
Add macOS command

Evernow committed:
Fix xeon naming mistakes

moviuro commented on products/intelprocessors.md:
```diff
@@ -3,7 +3,7 @@ permalink: /intel-processors
 iconSlug: intel
 category: device
-versionCommand: "'Get-CimInstance Win32_Processor' on Windows Powershell, 'cat /proc/cpuinfo' on Linux. Then check product classification on https://ark.intel.com/"
+versionCommand: "'Get-CimInstance Win32_Processor' on Windows Powershell, 'cat /proc/cpuinfo' on Linux, 'sysctl -n machdep.cpu.brand_string' on macOS. Then check product classification on https://ark.intel.com/"
```
`sysctl -n hw.model` on FreeBSD (and OpenBSD too, I guess, according to https://man.openbsd.org/sysctl.2#HW_MODEL~2)

Evernow replied:
Added with https://github.com/endoflife-date/endoflife.date/pull/1397/commits/c3746baf986006c4b151e955e14f47daea5c59fd , thank you!

Evernow committed:
Add FreeBSD/OpenBSD

Evernow committed:
Move commands to text body
